fastestimator/estimator.py

```diff
@@ -52,14 +53,14 @@ def __init__(self,
                  steps_per_epoch: Optional[int] = None,
                  traces: Union[Trace, Iterable[Trace]] = None,
                  log_steps: Optional[int] = 100,
-                 monitor_names: Optional[str] = None):
+                 monitor_names: Optional[Union[str, Set[str]]] = None):
```

Comment:
The type should be iterable because we should accept list as well.

we can convert it to set later.

Reply:
Done